src/backend/joanie/settings.py

```diff
@@ -401,6 +401,8 @@ class Development(Base):
     DEBUG = True
```

Comment:
in commit message:
they can be
overridden that pretty weird > overidden which is pretty weird ?

or which is disturbing